### PR TITLE
initialize sql array into platform files

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**
|      Q       |   A
|------------- | -----------
| Type         | feature
| BC Break     | no

#### Summary

initialize sql array into platform files

### CODE REVIEW

**Comment by AlessandroMinoccheri** [comment]
FIxed coding style violations @morozov

File: lib/Doctrine/DBAL/Platforms/DrizzlePlatform.php [hunk 1]
```diff
@@ -240,6 +240,7 @@ protected function _getCreateTableSQL($tableName, array $columns, array $options
         $query .= $this->buildTableOptions($options);
         $query .= $this->buildPartitionOptions($options);
 
+        $sql   = [];
         $sql[] = $query;
```
Comment (Ocramius) [inline lib/Doctrine/DBAL/Platforms/DrizzlePlatform.php]: Should really be `$sql = [$query]`

File: lib/Doctrine/DBAL/Platforms/MySqlPlatform.php [hunk 2]
```diff
@@ -457,6 +457,7 @@ protected function _getCreateTableSQL($tableName, array $columns, array $options
         $query .= $this->buildTableOptions($options);
         $query .= $this->buildPartitionOptions($options);
 
+        $sql    = [];
```
Comment (Ocramius) [inline lib/Doctrine/DBAL/Platforms/MySqlPlatform.php]: Same here: should really be `$sql = [$query]`

File: lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php [hunk 3]
```diff
@@ -782,6 +782,7 @@ protected function _getCreateTableSQL($tableName, array $columns, array $options
 
         $query = 'CREATE TABLE ' . $tableName . ' (' . $queryFields . ')';
 
+        $sql   = [];
         $sql[] = $query;
```
Comment (Ocramius) [inline lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php]: Same here: should really be `$sql = [$query]`

File: lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php [hunk 4]
```diff
@@ -304,6 +304,7 @@ protected function _getCreateTableSQL($tableName, array $columns, array $options
         }
         $query .= ')';
 
+        $sql   = [];
         $sql[] = $query;
```
Comment (Ocramius) [inline lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php]: Same here: should really be `$sql = [$query]`

File: lib/Doctrine/DBAL/Platforms/SqlitePlatform.php [hunk 5]
```diff
@@ -354,6 +354,7 @@ protected function _getCreateTableSQL($name, array $columns, array $options = []
             }
         }
 
+        $query   = [];
         $query[] = 'CREATE TABLE ' . $name . ' (' . $queryFields . ')';
```
Comment (Ocramius) [inline lib/Doctrine/DBAL/Platforms/SqlitePlatform.php]: Same here: should really be `$sql = [$query]`

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Review by Ocramius — DISMISSED** [review]
:+1:

**Comment by AlessandroMinoccheri** [comment]
Done @morozov

**Review by morozov — APPROVED** [review]

**Comment by morozov** [comment]
Thank you @AlessandroMinoccheri!

**Pull request merged**